recipes-test/ci-all-packages/ci-all-packages.bb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Felix Moessbauer <felix.moessbauer@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.
+# See COPYING.MIT file in the top-level directory.
+#
+
+# This is a meta recipe to test the building of all packages
+
+inherit dpkg-raw
+
+DEPENDS = "                      \
+    libedgetpu                   \
+    astunparse                   \
+    keras-preprocessing          \
+    opt-einsum                   \
+    pasta                        \
+    python-absl                  \
+    python-gast                  \
+    bazel-bootstrap              \
+    gasket-dkms                  \
+    gasket-module-${KERNEL_NAME} \
+    tensorflow                   \
+    pycoral                      \
+"
```

Comment:
You never add a proper newline at EOF - not a problem per se, just a bit ugly.

Reply:
Indeed. Can be fixed on next version.